src/query_loader/QueryLoader.cpp

```diff
@@ -208,6 +208,7 @@ InputQuery QueryLoader::loadQuery( const String &fileName )
         }
         else if ( coType == "max" )
         {
+            serializeConstraint += String( ",e,0,0" );
```

Comment:
It's on purpose - because when parsing an input query text file a new max constraint needs to be generated - and its constructor will search for the updated eliminated value (if exists). If we keep the updated serializeToString() method and run the code it will fail on processing the input query file.

Reply:
I still don't understand. The serialized constraint will already have the "e,1,17" or "e,0,0" suffix; why add another one?